egs/fisher_callhome_spanish/asr1/path.sh

```diff
@@ -0,0 +1,16 @@
+MAIN_ROOT=$PWD/../../..
```

Comment:
Please follow `path.sh` in the other recipes. We changed it recently #464

Reply:
Done